### PR TITLE
NumberPicker: increase test coverage in Karma runner

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1312
[Problem] NumberPicker: test coverage is too low
[Solution]
 - added new unit tests

obraz

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: tests/js/profile/wearable/widget/wearable/NumberPicker/NumberPicker.js [hunk 1]
```diff
@@ -50,10 +50,185 @@
...
+			widget._setValue(40);
+			assert.equal(element.value, 40, "Value has been properly set");
+			assert.equal(element.getAttribute("value"), "40", "Value has been properly set");
+
+			// case LOOP if value < min the value = max
+			helpers.stub(widget, "_updateValue", function (value) {
+				assert.ok(true, "_updateValue");
+				assert.equal(value, 60, "value is correct");
+			});
+			widget._setValue(10);
```
Comment (HunseopJeong) [inline tests/js/profile/wearable/widget/wearable/NumberPicker/NumberPicker.js:204]: Is it correct value?

Comment (TomaszLukawskiSam) [reply tests/js/profile/wearable/widget/wearable/NumberPicker/NumberPicker.js:204]: This test results from implementation of method (https://github.com/Samsung/TAU/blob/4470e5655a6416ea189945f8b8a2c1dbf5c05a07/src/js/profile/wearable/widget/wearable/NumberPicker.js#L374):
```
				// loop widget value;
				if (value > options.max) {
					value = options.min;
				} else if (value < options.min) {
					value = options.max;
				}
```

Comment (HunseopJeong) [reply tests/js/profile/wearable/widget/wearable/NumberPicker/NumberPicker.js:204]: Oh I see, it looks like the expected value.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**